solutions.js

```diff
@@ -1,5 +1,111 @@
 console.log('Defining functions!');
 
 function changeNavColor() {
-  // change the navbar to a different color
+  topbar.style['background-color'] = 'purple';
+
 };
+backgroundNav();
+
+var heading = document.querySelector('h1');
+  heading.style.fontSize = '25px';
+  heading.style['padding'] = '20px 0';
+  heading.prepend('This page is now under Nat\'s control!');
+
+  function removeImages() {
+    //links is an array of the child elements from links of the class cards
+    var links = [...document.querySelectorAll('.cards .image-list-link')];
+    //posts is an array of the child elements from posts of the class cards
+    var posts = [...document.querySelectorAll('.cards .post')];
+    //this checks the links array length, and the divides it by 2 to split the images in half.
+    var linkLength = links.length / 2;
+
+    //this iterates through each link to an image
+    for(var a = 0; a < linkLength.length; a++) {
+      //this tells the DOM to remove the child[links] from the parent[cards] at the index[a]
+       links[a].parentNode.removeChild(links[a]);
+       //this says that at index[a], set display to be hidden.
+       posts[a].style.display = 'none';
+    }
+  }
+
+  //Create a function that prompts the user to type "please" after clicking an image in order to see it larger.
+  //I'm going to create a variable to store the querySelector ELEMENT
+
+  function sayPlease() {
+    var image = [...document.querySelectorAll('.image-list-link img')];
+    //I want image to be an array because for each element in the image collection, I want something to happen when its clicked
+      image.forEach(function(element) {
+        //I want to start by creating a prompt to tell the user to type please.
+        var prompt = prompt(`I'm desperate for more destruction, so type please.`);
+        //for each element I'm iterating through, listen for a click
+          element.addEventListener(`click`, function(e){
```

Comment:
If the user doesn't type please, you'll want to stop the default event. Otherwise, you'll continue to change the final destination. 
```js
if (prompt !== 'please'){
  e.preventDefault();
} 
```